packages/next/build/index.ts

```diff
@@ -809,7 +809,7 @@ export default async function build(
     ) as BuildManifest
 
     const timeout = config.staticPageGenerationTimeout || 0
-    const sharedPool = config.experimental.sharedPool || false
+    const sharedPool = true
     const staticWorker = sharedPool
```

Comment:
Seems like more code can be removed since `sharedPool` is always true

Reply:
@styfle yes indeed, wanted to start with just removing the flag so that it can't be accidentally opted out of.